literature.bib

```diff
@@ -1,76 +1,116 @@
+@article{NST+:SoSyM21,
+	author = {Nieke, Michael and Sampaio, Gabriela and Th{\"u}m, Thomas and Seidl, Christoph and Teixeira, Leopoldo and Schaefer, Ina},
+	title = {{Guiding the Evolution of Product-Line Configurations}},
+	journal = SoSyM,
+	year = 2021,
+	note = {To appear}
+}
+
+@inproceedings{KTS:FormaliSE21,
+	author = {Kn\"{u}ppel, Alexander and Th\"{u}m, Thomas and Schaefer, Ina},
+	title = {{GUIDO: Automated Guidance for the Configuration of Deductive Program Verifiers}},
+	booktitle = FormaliSE,
+	publisher = ACM,
+	address = NY,
+	year = 2021,
+	note = {To appear}
+}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+@proceedings{TCA:SPLC21,
+  title={{Fourth International Workshop on Languages for Modelling Variability (MODEVAR@SPLC 2021)}},
+  editor={Th{\"{u}}m, Thomas and Collet, Philippe and Acher, Mathieu},
+  booktitle= SPLC,
+  isbn = {9781450384698},
+  doi = {10.1145/3461001.3473056},
+  publisher= ACM,
+  address = NY,
+  location = {Leicester, United Kingdom},
+  pages={205--205},
+  numpages = {1},
+  month = SEP,
+  year= 2021
+}
+
 @inproceedings{HST:SPLC21,
   author = {He{\ss}, Tobias and Sundermann, Chico and Th{\"u}m, Thomas},
   title = {{On the Scalability of Building Binary Decision Diagrams for Current Feature Models}},
   booktitle = SPLC,
   publisher = ACM,
+  pages = {131--135},
+  numpages = {5},
+  isbn = {9781450384698},
+  doi = {10.1145/3461001.3474452},
   address = NY,
-  note = {To appear},
+  location = {Leicester, United Kingdom},
```

Comment:
Introduce variable?

Reply:
Having a variable would be nice here because I also added an SPLC paper to my branch. I would use the variable then too. You can merge afterwards.